packages/controller-utils/src/index.test.ts

```diff
@@ -0,0 +1,75 @@
+import * as allExports from '.';
+
+describe('@metamask/controller-utils', () => {
+  it('has expected JavaScript exports', () => {
+    expect(Object.keys(allExports)).toMatchInlineSnapshot(`
+      Array [
+        "createServicePolicy",
```

Comment:
This test file did not previously exist. This line specifically refers to the new export we are adding; the rest already exist.